lib/elixir/lib/map.ex

```diff
@@ -15,7 +15,7 @@ defmodule Map do
 
   Maps do not impose any restriction on the key type: anything can be a key in a
   map. As a key-value structure, maps do not allow duplicated keys; keys are
-  compared using the match operator (`===`).
+  compared using the exact equality operator (`===`).
```

Comment:
Just a very very small nitpick: what if we use "exact-equality operator" here (with the hyphen)? It seems more correct but I'm not sure. Wdyt?